cdbot/cogs/help.py

```diff
@@ -0,0 +1,69 @@
+from discord import Colour, Embed
+from discord.ext import commands
+
+
+class EmbeddedHelpCommand(commands.HelpCommand):
+    def __init__(self):
+        super().__init__(command_attrs={'help': 'Gives detailed information about a command.'})
+
+    async def send_bot_help(self, mapping):
+        ctx = self.context
+        embed = Embed(
+            color=Colour.blue(),
+            description=f"Type ``{self.clean_prefix}help [command]`` for more info on a command.\n""You can also type "
+                        f"``{self.clean_prefix}help [category]`` for more info on a category."
+        )
+        for cog in mapping.keys():
+            if cog is not None:
```

Comment:
Could you use `if cog` instead of `if cog is not None` here?

Reply:
yes...yes i could have